cdc/sink/flowcontrol/flow_control_test.go

```diff
@@ -44,10 +39,8 @@ func (c *mockCallBacker) cb() error {
 	return c.injectedErr
 }
 
-func (s *flowControlSuite) TestMemoryQuotaBasic(c *check.C) {
-	defer testleak.AfterTest(c)()
-
-	controller := NewTableMemoryQuota(1024)
+func TestMemoryQuotaBasic(t *testing.T) {
+	controller := newTableMemoryQuota(1024)
```

Comment:
Maybe we should run the tests in parallel.

Reply:
Addressed. Thanks!